### ISSUE TITLE
[Babylon XR] The windows are frozen after exiting immersive mode

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [MozillaReality/FirefoxReality#3472]
**[Affected Versions]:**
- Firefox Reality version RC 10.2 (699262e).

**[Affected Platforms]:** 
- Oculus Quest - Android 7.1.1
- HTC Vive Focus - Android 7.1.2
- Pico Neo2 - Android 8.1.0

**[Prerequisites]:**
- The Firefox Reality browser is open.

**[Steps to reproduce]:**
1. Navigate to https://doc.babylonjs.com/how_to/webxr_demos_and_examples
2. Play a webxr example in immersive mode.
3. Exit immersive mode and interact whit the window.
4. Observe the behavior.

**[Expected results]:**
- The window is responding.

**[Actual results]:**
- The window is frozen.

**[Notes]:**
- The newly created windows are also affected by this issue, the content is not loading.
- After quiting and resuming the app the windows are no longer frozen.
- Attached is a screen recordings of the issue:
	- Screen Recording: [link](https://drive.google.com/file/d/1B2kwdjEoCNRj-QkM17a6ZzqbH9w00rim/view?usp=sharing).

**Comment by Softvision-RemusDranca** [comment]
This issue is also reproducible with https://playcanv.as/p/TAYVQgU2/ example but only after entering and exiting VR immersive 2 times.
@MortimerGoro should I log a new issue for this example or do both of them have the same root cause?

**Comment by MortimerGoro** [comment]
@Softvision-RemusDranca I'll test the playcanvas link and let you know

**Comment by MortimerGoro** [comment]
This a profiler capture which shows where Gecko is frozen: https://profiler.firefox.com/public/112813479d5a24f2c0072aaa26594560d4d084d7/calltree/?globalTrackOrder=0-1&hiddenLocalTracksByPid=10990-0~11056-0-1&localTrackOrderByPid=10990-2-0-1~11056-3-0-1-2~&thread=3&v=4

It looks like a WebGL layer issue

**Comment by Softvision-GeluHaiduc** [comment]
I have verified this issue with the latest Firefox Reality version 11-141760226 - 71aabd05 installed on the HTC Vive Focus Plus, Pico Neo2 and Pico G2 4K headsets.
Now, the windows are not frozen after exiting immersive mode from the examples on babylonjs.com.
